src/XrdPlugins.cmake

```diff
@@ -9,6 +9,7 @@ set( LIB_XRD_PSS        XrdPss-${PLUGIN_VERSION} )
 set( LIB_XRD_GPFS       XrdOssSIgpfsT-${PLUGIN_VERSION} )
 set( LIB_XRD_ZCRC32     XrdCksCalczcrc32-${PLUGIN_VERSION} )
 set( LIB_XRD_SSI        XrdSsi-${PLUGIN_VERSION} )
+set( LIB_XRD_SSILOG     XrdSsiLog-${PLUGIN_VERSION} )
```

Comment:
I tried to build this branch and it almost works but it does not install libXrdSsiLog-4.so plugin, I believe you (or @abh3) need to add `${LIB_XRD_SSILOG}` to `install(...)` below.